scene/resources/world_boundary_shape_2d.cpp

```diff
@@ -79,7 +79,7 @@ void WorldBoundaryShape2D::draw(const RID &p_to_rid, const Color &p_color) {
 
 	Vector2 l1[2] = { point - get_normal().orthogonal() * 100, point + get_normal().orthogonal() * 100 };
 	RS::get_singleton()->canvas_item_add_line(p_to_rid, l1[0], l1[1], p_color, 3);
-	Vector2 l2[2] = { point, point + get_normal() * 30 };
+	Vector2 l2[2] = { point + get_normal().normalized() * 1.5, point + get_normal() * 30 };
```

Comment:
To make it clearer I'd replace these `3` with a variable like `real_t line_width = 3.0;` and in here:
```suggestion
	Vector2 l2[2] = { point + get_normal().normalized() * (0.5 * line_width), point + get_normal() * 30 };
```